pkg/kafka/topicprovisioner.go

```diff
@@ -233,13 +249,25 @@ func (p *topicProvisioner) DeProvision(ctx context.Context, topics ...string) er
 		return nil
 	}
 
+	topicsToDelete := make([]string, 0, len(topics))
 	for _, topic := range topics {
 		if topic == "" {
-			return fmt.Errorf("invalid topic name %q", topic)
+			p.logger.Warn("skip topic: empty topic name", "topic", topic)
```

Comment:
why can this be empty?

Reply:
Normally it should not be, but nothing prevents one to make call like:

```go
proivisioner.DeProvision(ctx, []string{"topic-1", "", "topic-3"})
```

I wanted to make sure that empty topic names are at least logged, as it may indicate an issue in the code.